Source/Core/DolphinQt/Config/Mapping/GCPadEmu.cpp

```diff
@@ -3,14 +3,22 @@
 
 #include "DolphinQt/Config/Mapping/GCPadEmu.h"
 
+#include <string>
+
+#include <QApplication>
+#include <QCheckBox>
+#include <QDoubleSpinBox>
 #include <QGridLayout>
 #include <QGroupBox>
+#include <QLabel>
+#include <QPushButton>
 
 #include "Core/HW/GCPad.h"
 #include "Core/HW/GCPadEmu.h"
-
 #include "InputCommon/ControllerEmu/Setting/NumericSetting.h"
+#include "InputCommon/ControllerInterface/DInput/DInputKeyboardMouse.h"
 #include "InputCommon/InputConfig.h"
+#include "RightClickButton.h"
```

Comment:
We use relative paths for specifying headers:

```suggestion
#include "DolphinQt/QtUtils/RightClickButton.h"
```